Sources/SimpleSalesforceSDK/Models/SFAuth.swift

```diff
@@ -91,7 +91,11 @@ class SFAuth {
     "&client_id=\(config.clientId)" +
     "&refresh_token=\(refreshToken)"
     
-    guard let url = URL(string: "\(config.host)/oauth2/token") else { return }
+    guard let url = URL(string: "\(config.host)services/oauth2/token")
```

Comment:
What if the host supplied does not end with `/`?

I found this SO answer https://stackoverflow.com/questions/17419460/should-i-save-a-path-with-or-without-a-trailing-slash-at-the-end-whats-the-con

If we expect something lets make sure we either validate it at the time of configure call or actually modify the incoming url and add the trailing slash if the sdk user missed it